ssvsigner/go.mod

```diff
@@ -26,7 +26,7 @@ require (
 	github.com/microsoft/go-crypto-openssl v0.2.9
 	github.com/prysmaticlabs/go-bitfield v0.0.0-20240618144021-706c95b2dd15
 	github.com/ssvlabs/eth2-key-manager v1.5.2
-	github.com/ssvlabs/ssv v1.2.1-0.20250623120703-7787f963c4ad
+	github.com/ssvlabs/ssv v1.2.1-0.20250619015436-127dc5706a1b
```

Comment:
It seems to have become older. It used to be from June 23rd, and now it's from June 19th. Can we align it with the last `ssv` version?

Reply:
I haven't updated it, thanks!